presto-kafka/src/test/java/com/facebook/presto/kafka/util/EmbeddedKafka.java

```diff
@@ -121,9 +123,10 @@ public void createTopics(int partitions, int replication, Properties topicProper
         checkState(started.get() && !stopped.get(), "not started!");
 
         ZkClient zkClient = new ZkClient(getZookeeperConnectString(), 30_000, 30_000, ZKStringSerializer$.MODULE$);
+        ZkUtils zkUtils = ZkUtils.apply(zkClient, false);
```

Comment:
Curious: Looks like this variable is only used once. Can we inline it ?

Reply:
done